_posts/2024-01-02-resource-recommendations.html

```diff
@@ -0,0 +1,200 @@
+---
+author:
+duration: 4 Min Read
+excerpt: These resources have also formed and shaped Upstream. We’ve allowed these books to influence how we think and approach ministry.
+image: articles/feature-book-shelf.jpg
+image_alt: A shelf of books.
+layout: article
+nav_state: transparent-dark
+permalink: /articles/resource-recommendations/
+title: Resource Recommendations
+---
+
+<h2>Learn With Us</h2>
+
+<p>
+  There is an ocean of resources about missions, technology, faith and work,
+  theology…the list of topics goes on and on. If you’re like us, life is busy
+  and you can’t read everything. We’ve curated a shortlist that we hope
+  illuminates a trusted pathway through the sea of options. Or at least it can
+  serve as a good starting point.
+</p>
+
+<p>
+  These resources have also formed and shaped Upstream. We’ve allowed these
+  books to influence how we think and approach ministry. In this sense, the list
+  also gives a bit of a behind the scenes look into who we are and what we
+  value.
+</p>
+
+<div class="row justify-content-between align-items-center">
+  <div class="start-image-box order-1 col-12 col-md-5 text-center">
+    <img src="{{ "/assets/images/articles/resource_the-five-disfunctions.jpeg" |
+    relative_url }}" class="img-fluid resource-img" alt="The Five Dysfunctions of
+    a Team: A Leadership Fable">
+  </div>
+  <div class="content-box order-2 col-12 col-md-7 resource-detail">
+    <p class="resource-topic">Team DNA and Community</p>
```

Comment:
This should use overline font styling.
![Screenshot 2024-04-17 at 9 47 06 AM](https://github.com/UpstreamCode/upstreamcode.github.io/assets/31777263/ac55d141-563d-4470-a619-04eb0498fd86)

vs
<img width="435" alt="Screenshot 2024-04-17 at 9 48 12 AM" src="https://github.com/UpstreamCode/upstreamcode.github.io/assets/31777263/fa46c819-758c-4b9a-a38e-d8bedf7d9390">

Reply:
Same comments apply to other resources